src/hooks/index.ts

```diff
@@ -869,3 +869,10 @@ export const useResync = (wallet: YoroiWallet, options?: UseMutationOptions<void
     resync: mutation.mutate,
   }
 }
+
+export const useNfts = (wallet: YoroiWallet) => {
```

Comment:
I'd like to cache the native assets metadata at a cardano key level in the storage I.e `/implementation-id/cardano/<fingerprint>: metadata`